modules/configuring-private-storage-endpoint-azure-user-provided-vnet-subnet.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-private-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-private-storage-endpoint-azure-user-provided-vnet-subnet_{context}"]
+= Configuring a private storage endpoint on Azure with user-provided VNet and subnet names
+
+Use the following procedure to configure a storage account that has public network access disabled and is exposed behind a private storage endpoint on Azure.
+
+.Prerequisites
+
+* You have configured the image registry to run on Azure
+* You must know the VNet and subnet names used for your Azure environment.
+* If your network was configured in a separate resource group in Azure, you must also know its name.
+
+.Procedure 
+
+. Edit the Image Registry Operator `config` object and configure the private endpoint using your VNet and subnet names:
++
+[source,terminal]
+----
+$ oc edit configs.imageregistry/cluster
+----
++
+[source,terminal]
+----
+# ...
+spec:
+  [...]
+   storage:
+      azure:
+        [...]
+        networkAccess:
+          type: Internal
+          internal:
+            subnetName: <subnet_name>
+            vnetName: <vnet_name>
+            networkResourceGroupName: <network_resource_group_name>
+# ...
+----
+
+. Optional. Enter the following command to confirm that the Operator has completed provisioning. This might take a few minutes. 
++
+[source,terminal]
+----
+$ oc get configs.imageregistry/cluster -o=jsonpath="{.spec.storage.azure.privateEndpointName}" -w
+----
++
+[NOTE]
+====
+When redirect is enabled, pulling images from outside of the cluster will not work.
+====
+
+.Verification
+
+Use the following verification steps to ensure that users within the same cluster can pull images from the registry.
```

Comment:
I don't think we normally want a lead-in sentence before steps

https://redhat-documentation.github.io/supplementary-style-guide/#lead-in-sentences

Doesn't specify verification steps, but I assume that would be the same as procedure guidelines

Same comment for other modules